js-rattler/src/esm.ts

```diff
@@ -5,5 +5,6 @@ import mod from "../pkg/js_rattler_bg.wasm";
 //@ts-ignore
 import { initSync } from "../pkg/js_rattler";
 
-//@ts-ignore
-await initSync({ module: await mod() });
+export function initRattler() {
+    initSync({ module: mod() });
```

Comment:
I think the `mod` function here returns a Promise so it must be awaited.

Reply:
I think it's synchronous after we rollup it.